wordpress-mysql-replication/azuredeploy.json

```diff
@@ -0,0 +1,378 @@
+            "type": "string",
+            "metadata": {
+                "description": "wordpress site name"
+            }
+        },
+        "hostingPlanName": {
+            "type": "string",
+            "metadata": {
+                "description": "website host plan"
+            }
+        },
+        "hostingEnvironment": {
+            "type": "string",
+            "defaultValue": "",
+            "metadata": {
+                "description": "website host environment"
+            }
+        },
+        "sku": {
+            "type": "string",
+            "allowedValues": [
+                "Free",
+                "Shared",
+                "Basic",
+                "Standard",
+                "Premium"
+            ],
+            "defaultValue": "Standard",
+            "metadata": {
+                "description": "website sku"
+            }
+        },
+        "workerSize": {
+            "type": "string",
+            "allowedValues": [
+                "0",
+                "1",
+                "2"
+            ],
+            "defaultValue": "0",
+            "metadata": {
+                "description": "website worker size"
+            }
+        },
+        "webDeploy_AuthenticationKey": {
+            "type": "string",
+            "metadata": {
+                "description": "webdeploy authentication key"
+            }
+        },
+        "webDeploy_SecureAuthenticationKey": {
+            "type": "string",
+            "metadata": {
+                "description": "webdeploy secure authentication key"
+            }
+        },
+        "webDeploy_LoggedInKey": {
+            "type": "string",
+            "metadata": {
+                "description": "webdeploy logged in key"
+            }
+        },
+        "webDeploy_NonceKey": {
+            "type": "string",
+            "metadata": {
+                "description": "webdeploy nonce key"
+            }
+        },
+        "webDeploy_AuthenticationSalt": {
+            "type": "string",
+            "metadata": {
+                "description": "webdeploy authentication salt"
+            }
+        },
+        "webDeploy_SecureAuthenticationSalt": {
+            "type": "string",
+            "metadata": {
+                "description": "webdeploy secure authentication salt"
+            }
+        },
+        "webDeploy_LoggedInSalt": {
+            "type": "string",
+            "metadata": {
+                "description": "webdeploy logged in salt"
+            }
+        },
+        "webDeploy_NonceSalt": {
+            "type": "string",
+            "metadata": {
+                "description": "webdeploy nonce salt"
+            }
+        },
+        "dnsName": {
+            "type": "string",
+            "metadata": {
+                "description": "Connect to your cluster using dnsName.location.cloudapp.azure.com"
+            }
+        },
+        "wpDbUsername": {
+            "type": "string",
+            "defaultValue": "wpuser",
```

Comment:
Please remove the default username and let the user specify it as a security best practice